Microsoft.Research/ManagedContract.Setup/buildMSI10.xml

```diff
@@ -370,8 +371,33 @@
 
     ==========================================================-->
   <Target Name="MergeExes"
-    DependsOnTargets="ILMergeFoxtrot;ILMergeAsmMeta;ILMergeClousot;ILMergeCCDoc"/>
+    DependsOnTargets="PrepareMerges;ExecuteMerges"/>
 
+  <Target Name="PrepareMerges" BeforeTargets="ExecuteMerges"
+    DependsOnTargets="RestoreToolPackages;PrepareILMerge"/>
+	
+  <Target Name="RestoreToolPackages">
+    <Exec Command="&quot;$(SolutionDir).NuGet\NuGet.exe&quot; restore &quot;$(SolutionDir)packages.config&quot; -PackagesDirectory &quot;$(SolutionDir)packages&quot;" />

```

Comment:
The MSBuild-Integrated package restore will be migrated to automatic package restore as part of a separate commit. At least the VSIX projects will rely on command-line package restore before the build to download packages which integrate into the build process, so we're going to have to run the restore from the .bat file eventually anyway. I prefer to keep it consistent even if running the restore within MSBuild would not cause problems for some subset of the projects.